FNPlugin/Wasteheat/FNRadiator.cs

```diff
@@ -334,9 +335,13 @@ public double EffectiveRadiatorArea
 
                 effectiveRadiativeArea = PluginHelper.RadiatorAreaMultiplier * areaMultiplier * radiatorArea;
 
-                return hasSurfaceAreaUpgradeTechReq 
-                    ? effectiveRadiativeArea * surfaceAreaUpgradeMult 
-                    : effectiveRadiativeArea; 
+                // Because I have absolutely no idea what I'm doing, I'm taking some short cuts and major simplifications.
+                // This is the radius of a circular radiator, (operating in a vacuum)
+                sphericalCowInAVaccum = (effectiveRadiativeArea / Mathf.PI).Sqrt();
```

Comment:
If it is a circular radiator, why is it called spericalCow? (cylindrical maybe?)

Reply:
From the lead-in, one would expect a spherical horse.

Reply:
https://en.wikipedia.org/wiki/Spherical_cow - "A spherical cow is a humorous metaphor for highly simplified scientific models of complex real life phenomena."

Compare and contrast the recent addition where reactors have a defined radiatorArea size of 0 - the code calculates the surface area of the reactor at run time. 

While this code could have taken a similar approach, and calculated the radius along the x, y and z axis, and calculated appropriate movements there, it would have taken far more familiarity with Unity, graphics, meshes, and probably maths than I have (and resulting time learning how to do all of those things). By simplifying the approach, I was able to come up with a problem that I could come up with a solution for it.

Reply:
it that case it would be useful the configure it externally. I believe you already added a public property for this. Then it's just a matter of adding a simple != 0 condition to prevent it from taking a generated value.